Engine/BuiltInShaders/common/Material.sh

```diff
@@ -110,15 +110,16 @@ Material GetMaterial(vec2 uv, vec3 normal, mat3 TBN) {
 	material.roughness = orm.y;
 	material.metallic = orm.z;
 #else
-	material.roughness = u_metallicRoughnessFactor.y;
-	material.metallic = u_metallicRoughnessFactor.x;
+	material.roughness = u_metallicRoughnessRefectanceFactor.y;
+	material.metallic = u_metallicRoughnessRefectanceFactor.x;
 #endif
 
 #if defined(EMISSIVEMAP)
 	material.emissive = SampleEmissiveTexture(uv);
 #endif
 	
-	material.F0 = mix(vec3_splat(0.04), material.albedo, material.metallic);
+	float refectance = u_metallicRoughnessRefectanceFactor.z;
+	material.F0 = mix(0.16 * refectance * refectance, material.albedo, material.metallic);
```

Comment:
Since there are no materials in the real world with F0 below 0.02, the reflectance parameter should not be lower than $\sqrt{\flac{0.02}{0.16}}\approx 0.353553390$